### DISCUSSION TITLE
Conditional Results Based on Viewport/Screen Size in Tag Pairs

### DISCUSSION THREAD

**Discussion by dugtratch** [statamic/cms#10402] [Q&A] [answered]
Newb question here:

Is there a way to create conditional results based on viewport (eg mobile vs desktop)?  

For example, from the docs:

`{{ collection:blog limit="5" }}
    <a href="{{ url }}">{{ title }}</a>
{{ /collection:blog }}`

Can I get 3 on mobile and 5 on larger devices? Or:

`{{ collection:blog paginate="10" as="posts" }}
---
{{ /collection:blog }}`

set the paginate to different values?

**Comment by jackmcdade** [comment, marked as answer]
Alas no, server side isn't going to know the viewport. 

What I would do is pull 5 and then hide the last 2 on mobile with CSS.

**Reply by dugtratch** [reply]
All good. I just wanted to be sure there wasn't something I was missing.


Thanks for the blazing fast reply!
